src/components/RecordingSummary.vue

```diff
@@ -0,0 +1,272 @@
+<template>
+  <a
+    :href="`/recording/${item.id}`"
+    class="recording-summary"
+    @click="event => navigateToRecording(event, item.id)"
+  >
+    <div class="recording-type">
+      <span v-if="item.type === 'audio'">
+        <font-awesome-icon
+          :icon="['far', 'file-audio']"
+          size="2x"
+        />
+      </span>
+      <span v-else-if="item.type === 'thermalRaw'">
+        <font-awesome-icon
+          :icon="['far', 'file-video']"
+          size="2x"
+        />
+      </span>
+    </div>
+    <div class="recording-main">
+      <div class="recording-details">
+        <span class="recording-group">
+          <font-awesome-icon
+            icon="users"
+            size="xs"
+          />
+          {{ item.groupname }}
+        </span>
+        <span class="recording-device">
+          <font-awesome-icon
+            icon="microchip"
+            size="xs"
+          />
+          {{ item.devicename }}
+        </span>
+      </div>
+      <div
+        v-if="item.tags.length !== 0"
+        class="recording-tags"
+      >
+        <span
+          v-for="(tag, index) in item.tags"
+          :key="index"
+          :class="[
+            'tag',
+            'badge',
+            tag.class
+          ]"
+        >
+          <!-- TODO: Change icon when info is available from API about whether tag was applied by human or AI -->
```

Comment:
There should already be an `automatic` field on each tag to indicate whether it's AI generated or human generated